_posts/2022-11-20-Ripasso-release-0-6-0.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Release of Ripasso version 0.6.0"
+author: capitol
+category: infrastructure
+---
+![ripasso-cursive](/images/ripasso-cursive-0.4.0.png)
+
+Time passes as water under a bridge, it is yet again to release a version of ripasso. We present version 0.6.0.
```

Comment:
perhaps it should be "it is yet again time to"

Reply:
fixed